### PR TITLE
add x-state and reanimated

### PR DESCRIPTION
**Pull request by NamanRajputXXI**
- [x] add x-state in Meway-table
- [x] Make minimal ui

### CODE REVIEW

File: apps/meway-tablet/src/components/GlobalState.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+// import React, { createContext } from "react";
```
Comment (sutyum) [inline apps/meway-tablet/src/components/GlobalState.tsx:1]: Remove this file if it is not needed

File: apps/meway-tablet/src/machines/navigationMachine.typegen.ts [hunk 2]
```diff
@@ -0,0 +1,71 @@
+// // This file was automatically generated. Edits will be overwritten
```
Comment (sutyum) [inline apps/meway-tablet/src/machines/navigationMachine.typegen.ts:1]: remove if not needed

**Review by sutyum — CHANGES_REQUESTED** [review]
need to remove `.idea` directory.

File: apps/meway-tablet/src/machines/navigationMachine.typegen.ts (hunk 2 above)
Comment (NamanRajputXXI) [reply apps/meway-tablet/src/machines/navigationMachine.typegen.ts]: removed

File: apps/meway-tablet/src/components/GlobalState.tsx (hunk 1 above)
Comment (NamanRajputXXI) [reply apps/meway-tablet/src/components/GlobalState.tsx]: Removed

**Review by sutyum — APPROVED** [review]

**Pull request merged**